legend-engine-language-pure-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/to/data/ModelStoreDataGrammarComposer.java

```diff
@@ -95,22 +96,31 @@ public String compose(ModelStoreData data)
                     str.append(context.getIndentationString());
                     str.append(type).append(":\n");
 
-                    if (((Collection) data.instances.get(type)).values.size() == 1)
+                    if (!(data.instances.get(type) instanceof Pair))
```

Comment:
do we really need to wrap, can we not have just if else blocks?

Reply:
model store data with pair needs to be handled separately

Reply:
Resolved